Recrutify/Recrutify.Host/Startup.cs

```diff
@@ -155,7 +156,7 @@ public void Configure(IApplicationBuilder app, VersionedODataModelBuilder modelB
             }
 
             app.UseCors(Constants.Cors.CorsForUI);
-
+            app.Use((context, next) => { context.Request.Scheme = "https"; return next(); });
```

Comment:
Not correct to change scheme manually, there are correct existing mechanisms for solving such problems